org-formation/060-cloud-trail/README.md

```diff
@@ -0,0 +1,6 @@
+### Purpose of these templates
+The templates in this folder enable [CloudTrail](https://docs.aws.amazon.com/awscloudtrail/latest/userguide/cloudtrail-user-guide.html) in each account. CloudTrail is an essential service that implements an audit log of all actions taken within the account.
+
+The (audit) trail logs are stored in a central S3 bucket in the LogArchive account, which is in the Shared OU and therefore considered a production account. Analytics can take place based on the logs in that S3 buckets, for example, based on [S3 notifications](https://docs.aws.amazon.com/AmazonS3/latest/dev/NotificationHowTo.html).
```

Comment:
Is this accurate? That the Shared OU defines production?

Reply:
this is just a general doc i copied from [ofn reference project](https://github.com/org-formation/org-formation-reference/tree/master/src/templates/060-cloud-trail).  we don't have a `Shared OU` at this time, so this really has no meaning for us.